### DISCUSSION TITLE
API ADDONS

### DISCUSSION THREAD

**Discussion by PedroMPagani** [BG-Software-LLC/SuperiorSkyblock2#304] [Denied Feature Requests] [answered]
Hey, i'm writing some to the  SSB-SlimeWorldManager,
can you implement cancellable to the IslandSchematicPasteEvent? so i can load the world from a pattern and reduce all of this resource usage from the schematic, please

**Comment by OmerBenGera** [comment, marked as answer]
No, there are other ways to do what you want.
